code/game/machinery/suit_storage_unit.dm

```diff
@@ -275,6 +279,26 @@
 
 /obj/machinery/suit_storage_unit/Initialize()
 	. = ..()
+
+	component_parts = list()
+	component_parts += new board_type(null)
+	component_parts += new /obj/item/stock_parts/matter_bin(null)
+	component_parts += new /obj/item/stock_parts/micro_laser(null)
+	component_parts += new /obj/item/stack/cable_coil(null,3)
+	if(board_type == /obj/item/circuitboard/suit_storage_unit/industrial)
+		component_parts += new /obj/item/stack/sheet/plasteel(null)
+		component_parts += new /obj/item/stack/sheet/plasteel(null)
+		component_parts += new /obj/item/stack/sheet/plasteel(null)
+		component_parts += new /obj/item/stack/sheet/plasteel(null)
+		component_parts += new /obj/item/stack/sheet/plasteel(null)
+	else
+		component_parts += new /obj/item/stack/sheet/rglass(null)
+		component_parts += new /obj/item/stack/sheet/rglass(null)
+		component_parts += new /obj/item/stack/sheet/rglass(null)
+		component_parts += new /obj/item/stack/sheet/rglass(null)
+		component_parts += new /obj/item/stack/sheet/rglass(null)
+	RefreshParts() // MMTODO: needs "upgrades" for the different levels of parts.
```

Comment:
What does "MMTODO" means? I am curious

Reply:
If i write just "TODO" i have to search through all of em everywhere. Writing code at 2am and then wanting to sleep hit me.

Reply:
Wow the extenstion didnt work, anywho, see below.

Reply:
Update: not going to add any upgradeable bits for this, frankly. More an aesthetic/being able to move stuff around PR anyway.